library/std/build.rs

```diff
@@ -48,7 +54,7 @@ fn main() {
         // - mipsel-sony-psp
         // - nvptx64-nvidia-cuda
         // - arch=avr
-        // - JSON targets
+        // - JSON targets not describing an excluded target above.
```

Comment:
```suggestion
        // - JSON targets
```

I think just undoing this change in "docs" makes the most sense.

Reply:
Ah gotcha, I misread, sorry!